src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costOneDay = 40;
+  const fullPrice = days * costOneDay;
+
+  if (days < 3) {
+    return fullPrice;
+  } else if (days < 7) {
+    return fullPrice - 20;
+  } else {
+    return fullPrice - 50;
+  }
```

Comment:
You can delete last else:
```suggestion
  return fullPrice - 50;
```